docs/releases/1.11.txt

```diff
@@ -254,6 +254,9 @@ Forms
   <django.forms.Form.get_initial_for_field>` method returns initial data for a
   form field.
 
+* The ``PasswordResetForm.get_users()`` method now supports a custom email
```

Comment:
PasswordResetForm supports custom user models that use an email field named something other than 'email'. Set `CustomUser.EMAIL_FIELD` to the name of the field.
(add links as appropriate)